README.md

```diff
@@ -67,6 +67,37 @@ This will create and run a bot which always reply "Hello World" in the console.
 You'll notice that there is an async function to be passed to the `onEvent`
 method, we call it handler. You can put your bot logic in there.
 
+## Notable Features
+
+### Messenger
+
+* Messenger Profile Sync
+* Attachment Upload
+* Handover Protocol
+* Targeting Broadcast Messages
+* Built-in NLP
+* Multiple Pages
+
+### LINE
+
+* Reply, Push, Multicast
```

Comment:
We didn't export multicast on `context` (since they need array of IDs) but it can be still used with `context.client.multicastXXX`. Not sure if we should put it here.